src/nvim/msgpack_rpc/server.c

```diff
@@ -67,6 +68,30 @@ void server_teardown(void)
   GA_DEEP_CLEAR(&watchers, SocketWatcher *, close_socket_watcher);
 }
 
+/// Return unique address for local server.
+///
+/// In Windows this is a local pipe address in the format
+///     \\.\pipe\nvim-<PID>-<COUNTER>.
+///
+/// For other systems it is a path returned by vim_tempname().
+///
+/// This function is NOT thread safe
+///
+char *server_address(void)
+{
+#ifdef WIN32
+  static uint32_t count = 0;
+  char template[ADDRESS_MAX_SIZE] = "";
+  if (template[0] == '\0') {
```

Comment:
Leftover, `template` used to be a static buffer with the value being initialized only once. Removing the if and the initializer.